siteimprove/admin/js/siteimprove.js

```diff
@@ -50,6 +55,33 @@
         ]);
         return;
       }
+      _si.push(['onHighlight', function(highlightInfo) {
+        // Remove highlight tag wrapper
+        $(".si-highlight").contents().unwrap();
+        // Create an span tag for every highlight
+        $.each(highlightInfo.highlights, function(index, highlight) {
+          var $element = $(highlight.selector);
+          var text = $element.text();
```

Comment:
Are we sure the highlight is always a piece of text? Can't it be an image as well? I mean it could be an image with bad contrast or something...

Reply:
An ajustment was made at the commit e2ed1a36e8f9a710fb6ba85d3049f53e6264cd6d